modules/tinymce/src/themes/silver/test/ts/browser/editor/core/SimpleControlsTest.ts

```diff
@@ -0,0 +1,85 @@
+import { Chain, Log, UiFinder } from '@ephox/agar';
+import { Editor as McEditor, TinyApis } from '@ephox/mcagar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { SugarBody } from '@ephox/sugar';
+import Theme from 'tinymce/themes/silver/Theme';
+import Editor from 'tinymce/core/api/Editor';
+
+
+UnitTest.asyncTest('browser.tinymce.themes.silver.editor.core.SimpleControlsTest', (success, failure) => {
+  Theme();
+
+  const settings = {
+    base_url: '/project/tinymce/js/tinymce',
+    toolbar: 'bold italic underline strikethrough'
+  };
+
+  const sAssertToolbarButtonPressed = (title: string) => {
+    return UiFinder.sExists(SugarBody.body(), `button[title="${title}"][aria-pressed="true"]`);
+  };
+
+  Chain.pipeline([
+    McEditor.cFromSettings(settings),
```

Comment:
This should be using `TinyLoader.setup` as we're only ever initialising a single editor.

- [ ] Use TinyLoader instead here